packages/grafana-plugin-configs/webpack.config.ts

```diff
@@ -0,0 +1,208 @@
+import CopyWebpackPlugin from 'copy-webpack-plugin';
+import ESLintPlugin from 'eslint-webpack-plugin';
+import ForkTsCheckerWebpackPlugin from 'fork-ts-checker-webpack-plugin';
+import path from 'path';
+import ReplaceInFileWebpackPlugin from 'replace-in-file-webpack-plugin';
+import { Configuration } from 'webpack';
+
+import { DIST_DIR } from './constants';
+import { getPackageJson, getPluginJson, getEntries, hasLicense } from './utils';
+
+function skipFiles(f: string): boolean {
+  if (f.includes('/dist/')) {
+    // avoid copying files already in dist
+    return false;
+  }
+  if (f.includes('/tsconfig.json')) {
+    // avoid copying tsconfig.json
+    return false;
+  }
+  return true;
+}
+
+const config = async (env: any): Promise<Configuration> => {
+  const pluginJson = getPluginJson();
+  const baseConfig: Configuration = {
+    cache: {
+      type: 'filesystem',
+      buildDependencies: {
+        config: [__filename],
+      },
+    },
+
+    context: process.cwd(),
+
+    devtool: env.production ? 'source-map' : 'eval-source-map',
+
+    entry: await getEntries(),
+
+    externals: [
+      'lodash',
+      'jquery',
+      'moment',
+      'slate',
+      'emotion',
+      '@emotion/react',
+      '@emotion/css',
+      'prismjs',
+      'slate-plain-serializer',
+      '@grafana/slate-react',
+      'react',
+      'react-dom',
+      'react-redux',
+      'redux',
+      'rxjs',
+      'react-router',
+      'react-router-dom',
+      'd3',
+      'angular',
+      '@grafana/ui',
+      '@grafana/runtime',
+      '@grafana/data',
+
+      // Mark legacy SDK imports as external if their name starts with the "grafana/" prefix
+      ({ request }, callback) => {
+        const prefix = 'grafana/';
+        const hasPrefix = (request: any) => request.indexOf(prefix) === 0;
+        const stripPrefix = (request: any) => request.substr(prefix.length);
```

Comment:
the type of request is defined above as `string | undefined`. if this is typed correctly you'll see this will blow up if request is `undefined`...